R/fortify.r

```diff
@@ -19,9 +19,12 @@ fortify.NULL <- function(model, data, ...) waiver()
 fortify.function <- function(model, data, ...) model
 #' @export
 fortify.default <- function(model, data, ...) {
-  stop(
+  msg <- paste0(
     "ggplot2 doesn't know how to deal with data of class ",
-    paste(class(model), collapse = "/"),
-    call. = FALSE
+    paste(class(model), collapse = "/"), "."
   )
+  if (class(model) == "uneval") {
+    msg <- paste0(msg, " Did you forget the data argument in ggplot()?")
```

Comment:
I think that the error message will get called only if `ggplot()` has an `aes()` and no `data` argument:
```r
> ggplot() + geom_point(aes(x = x, y = y))
Error in FUN(X[[i]], ...) : object 'x' not found 
```
But 
```
> ggplot(aes(x = x, y = y)) + geom_point()
 Error: ggplot2 doesn't know how to deal with data of class uneval. Did you forget the `data` argument in ggplot()?
```